pkg/state/state_pod_security_policy.go

```diff
@@ -102,6 +104,13 @@ func (s *statePodSecurityPolicy) Sync(customResource interface{}, infoCatalog In
 // Get a map of source kinds that should be watched for the state keyed by the source kind name
 func (s *statePodSecurityPolicy) GetWatchSources() map[string]*source.Kind {
 	wr := make(map[string]*source.Kind)
+	psp := &policyv1beta1.PodSecurityPolicyList{}
+	err := s.client.List(context.TODO(), psp)
```

Comment:
Should we check for error type for NotMatch?

"k8s.io/apimachinery/pkg/api/meta"

meta.IsNoMatchError(err)

Reply:
Let's do it! I checked "k8s.io/apimachinery/pkg/api/errors" and it doesn't contain required logic. Thanks for the pointing me on it